pysys/basetest.py

```diff
@@ -884,21 +884,23 @@ def pythonDocTest(self, pythonFile, pythonPath=None, output=None, environs=None,
 		@param output: the output file; if not specified, '%s-doctest.txt' is used with 
 		the basename of the python file. 
 		
-		@param kwargs: extra arguments are passed to startProcess
+		@param kwargs: extra arguments are passed to startProcess/startPython. 
 		"""
 		assert os.path.exists(os.path.abspath(pythonFile)), os.path.abspath(pythonFile)
 		
 		if not output: output = '%s-doctest.txt'%os.path.basename(pythonFile).replace('.py','')
-		
-		p = self.startProcess(
-			sys.executable, 
+		self.log.info('Starting with env: %s', self.createEnvirons(overrides=[environs, {
```

Comment:
Was this logging supposed to be retained or just there for debugging during development?
If it's meant to stay in I don't like that the environment is being generated twice: once to log, once later to pass to startPython().

Reply:
good spot, that was a mistake. removed it